src/lib/tune/widgets.rb

```diff
@@ -0,0 +1,45 @@
+# encoding: utf-8
+
+# ------------------------------------------------------------------------------
+# Copyright (c) 2017 SUSE LLC
+#
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of version 2 of the GNU General Public License as published by the
+# Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, contact SUSE.
+#
+# To contact SUSE about this file by physical or electronic mail, you may find
+# current contact information at www.suse.com.
+# ------------------------------------------------------------------------------
+
+require "yast"
+require "cwm/widget"
+
+module Tune
+  module Widgets
+    class SystemInformation < CWM::PushButton
+      include Yast::Logger
+      def initialize
+        textdomain "tune"
+      end
+
+      def label
+        # PushButton label
+        _("System Information")
+      end
+
+      def handle
+        Yast::WFM.CallFunction("inst_hwinfo", [])
```

Comment:
The module can internally change some kernel settings, but AFAIK it does not report anything in the summary.